### ISSUE TITLE
start pooled 2012-2018 analysis on deenr node

### ISSUE THREAD

**Issue by katcatalano** [katcatalano/parentage#12]
1) all input files the same 2) all offspring and all parents
Start on amarel deenr node, and set it up so it outputs current results when it hits wall time and then re-start.

**Comment by katcatalano** [comment]
Had to stop the analysis referenced in the 4/23 comment because it didn't finish in time for the wall time out.

In commit 271719f I re-started the pooled analysis with 1340 loci and the new fish_indiv metadata used for sorting. I used a long run, low precision, and FL-PLS hybrid configuration to cut down on time as much as possible. This is less than ideal for accuracy, but I think that if I can compare the results to what I got with the full likelihood (FL), high precision annual runs, then I can see if all new parentage assignments are with fish that weren't ever recorded as offspring. Those should be the only new matches, and if the results aren't otherwise the same as the annual results then I can try to adjust the input parameters for colony to potentially use FL rather than the FL-PLS hybrid method.

@agdedrick the pooled run is started on amarel. I'll update as updates become available.
